glusterd2/utils/pprof.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"net/http"
+	"net/http/pprof"
+	runtime_pprof "runtime/pprof"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	basePath = "/debug/pprof/"
+)
+
+func addPath(router *mux.Router, name string, handler http.Handler) {
+	router.Path(basePath + name).Name(name).Handler(handler)
+	log.WithField("handler", handler).Debug("Starting profile for %s", name)
```

Comment:
@harigowtham can you please check how this log gets printed, I think you need to print basePath+Name

Reply:
Any variable should be part of log fields. Log message should be constant

Reply:
Have made the debug log message constant

> PR Looks good, please update developer nodes about using this feature

Do you want the developer note's update as a part of this patch or a different one?

Reply:
it good to have it with this PR. https://github.com/heketi/heketi/pull/1396 this will help you to document.

Reply:
its along with the example, will create a new file as mentioned and update it